scripts/umbrel-os/external-storage/mount

```diff
@@ -200,6 +200,19 @@ main () {
   mkswap "${SWAP_FILE}"
   swapon "${SWAP_FILE}"
 
+  echo "Checking if docker containers are on the SSD"
+  if [[ -f "${MOUNT_POINT}/docker" ]]; then
+    echo "Yes, docker data is on the SSD"
+  else 
+    echo "No, docker data is on the SD"
+    echo "Moving data..."
+    cp  --recursive \
+        --archive \
+        --no-target-directory \
+        /var/lib/docker ${MOUNT_POINT}/docker
+    mount --bind ${MOUNT_POINT}/docker /var/lib/docker
```

Comment:
Looks like this is only ever externally mounting the directory on the initial setup and will fall back to the SD card directory on subsequent boots.

We should move the bind mount line outside the conditional checks. Only the copy is conditional, we always want to run the bind mount.

Reply:
We should also do a quick double check the mount has succeeded. You can see how we do that above with `$UMBREL_ROOT`/`$SWAP_DIR`.

Reply:
Done.